src/core/templates-discovering.ts

```diff
@@ -0,0 +1,83 @@
+import Vue, { CreateElement } from "vue";
+import { IConfigurable } from "./configuration-component";
+import { ScopedSlot } from "vue/types/vnode";
+
+const TEMPLATE_PROP = "template";
+
+interface IEventBusHolder {
+    eventBus: Vue;
+}
+
+function asConfigurable(component: Vue): IConfigurable | undefined {
+    if (!component.$vnode) {
+        return undefined;
+    }
+
+    const configurable = component.$vnode.componentOptions as any as IConfigurable;
+    if (!configurable.$_config || !configurable.$_config.name) {
+        return undefined;
+    }
+
+    return configurable;
+}
+
+function hasTemplate(component: Vue) {
+    return TEMPLATE_PROP in component.$props && (component.$vnode.data && component.$vnode.data.scopedSlots);
+}
+
+function discover(component: Vue): Record<string, ScopedSlot> {
+    const templates = { ...component.$scopedSlots };
+    if (!!component.$slots["default"]) {
```

Comment:
Get rid of the condition. It will be more consistent to remove all default templates.